test/users/engin/partial_reduction_support/modules/dsiMethods.chpl

```diff
@@ -82,7 +82,7 @@ iter DefaultRectangularArr.dsiPartialThese(param onlyDim,
     otherIdx,
     param tag: iterKind) where tag == iterKind.standalone &&
       __primitive("method call resolves", dom, "dsiPartialThese",
-                                          onlyDim, otherIdx, tag=tag) {
+                                          onlyDim, otherIdx, tag) {
```

Comment:
@dlongnecke-cray - I thought we had a different primitive to do method call resolves where we can nest the call? I think using that here (and in the related cases in the same file) would be a better fix, since in principle the change implemented here so far could make some things not work that used to work.

Oh, wait, this is only in a test. So in that case I think it is OK either way as long as these specific tests work.

Even so it looks like @dlongnecke-cray already made the change I am talking about in the modules -- e.g.

https://github.com/chapel-lang/chapel/blob/ee4bae6752146a4c7c659dbc7bdf7068515b3671/modules/internal/ChapelArray.chpl#L1340

So I think it is best to mirror that change here.

Reply:
I changed all the `"method call resolves"` to just `"resolves"` in this file, if that's what you meant by 
>it is best to mirror that change here